<!--

  Copyright (C) 2018-2020 LEIDOS.

  Licensed under the Apache License, Version 2.0 (the "License"); you may not
  use this file except in compliance with the License. You may obtain a copy of
  the License at

  http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  License for the specific language governing permissions and limitations under
  the License.
-->
<!--
 
Primary launch file for platooning strategic plugin node
Loads parameters and configures logging for the node
 
-->
<launch>
<node pkg="platoon_control" type="platoon_control" name="platoon_control">
    <rosparam command="load" file="$(find platoon_control)/config/parameters.yaml" />
</node>
</launch>
